### ISSUE TITLE
adding a second show from active search results, causes display of trending results instead.

### ISSUE THREAD

**Issue by wethecom** [SchizoDuckie/DuckieTV#1097]
adding a show tab it jumps to favorites when browsing adding show results and when adding shows. it dosnt seem like it should, i have to keep relooking up shows and jump back and forth...make it stop please when im browsing search result it should stay there until i navigate away intentionally
DuckieTV Web Based | 1.1.5
-- | --
Current Date and Time | Wed, 13 Jun 2018 21:39:15 GMT
NWJS, Chromium | 0.25.0 , 61.0.3163.79
UserAgent | DuckieTV-Standalone v1.1.5 Windows NT 6.1; Win64; x64
Platform, Vendor | Win32, Google Inc.

**Comment by 2635599** [comment]
i've notice this behavior for at least the last couple of weeks, and yes it's annoying and needs to be fixed.

**Comment by Js41637** [comment]
I'm on the browser version and I've never noticed this behavior.

**Comment by garfield69** [comment]
@wethecom could you try the latest nightly standalone and see if it behaves the same way as you describe?
Take a precautionary duckietv backup first, using the *settings, backup* sidepanel, *backup database and settings* button.
Then download and install from https://github.com/DuckieTV/Nightlies/releases

**Comment by wethecom** [comment]
i updated to the latest build ..it is still a problem but now it take 3 adding shows before it navigates away
when you look up a show say gundam and there are 3 or 4 versions of the show...you press plus it adds the show then you press plus to another show boom it navigates away to your favorites and you have restart all over again to do the search and add the similar show.

**Comment by garfield69** [comment]
@Js41637  so the steps to reproduce the problem are:
1. navigate to the trending page
2. enter `gundam` in the search box and wait for the results
3. select and add a show from the results
4. then select or add a second show

at this point, the results are replaced by the default trending list, note that the search box still holds the original search term.
using the delete key and typing `m` restarts the search and restores the search results again.

so it looks like the search term is actually forgotten by the search processor?

**Comment by wethecom** [comment]
@Js41637 so the steps to reproduce the problem are:

*navigate to the add show page*
enter gundam in the search box and wait for the results
select and add a show from the results
then select or add a second show

**Comment by garfield69** [comment]
yeah no problem, the add-a-show page is the trakt.tv trending page, same place to us the devs ;-)

**Comment by wethecom** [comment]
ahhh i have no idea who im talking to

**Comment by Js41637** [comment]
why not both
